src/client/components/SuggestedRoutes/SuggestedRoutesList.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import SuggestedRouteCard from "./SuggestedRouteCard";
+
+export default class SuggestedRoutesList extends Component {
+  state = {
+    selectedId: null
+  };
+
+  handleClick = id => {
+    this.setState({
+      ...this.state,
+      selectedId: id
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        {this.props.list.map(i => {
```

Comment:
it's alway better to use more meaningful names, what is a list here? what is i refers to? it good naming make the code more understandable

Reply:
You mean "what is a list item here", right? Because I agree that `i` is not an idea variable name.